.github/workflows/java-pr-jobs.yml

```diff
@@ -19,3 +19,7 @@ jobs:
           java-version: 1.8
       - name: Test
         run: mvn test
```

Comment:
We've got duplication here. Also since test does a compile do we need both? Does the UI code add any integration tests? If so should do an `mvn verify`

Reply:
These tests are java only - no cross coverage (yet). I suspect rebase fun here - will update and check with you.